index.html

```diff
@@ -44,6 +43,7 @@
 
 </form>
 
+<script src="assests/javascript/app.js"></script>
```

Comment:
Add a comment to show what this is.  I always add something like <!-- External Scripting --> in the line above the script.